bins/rad-identities-dev/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "rad-identities-dev"
+version = "0.1.0"
+authors = ["Fintan Halpenny fintan.halpenny@gmail.com"]
```

Comment:
does cargo not want some angle brackets around the email address?

Reply:
Yes, apparently[0]

[0]: https://doc.rust-lang.org/cargo/reference/manifest.html#the-authors-field